Lib/asyncio/unix_events.py

```diff
@@ -680,7 +646,7 @@ def _start(self, args, shell, stdin, stdout, stderr, bufsize, **kwargs):
             # needed by close_fds=False on Python 3.3 and older
             # (Python 3.4 implements the PEP 446, socketpair returns
             # non-inheritable sockets)
-            _set_inheritable(stdin_w.fileno(), False)
+            os.set_inheritable(stdin_w.fileno(), False)
```

Comment:
> Does not the above comment make this unneeded? @vstinner, what are your thoughts?

Moved the question from #4633
@vstinner please review

Reply:
os.set_inheritable(stdin_w.fileno(), False) is useless on Python 3.4 and newer. Remove the call with its comment.

Reply:
If you don't trust me, please trust the tests ;-)
https://github.com/python/cpython/blob/ef83806f5ec01f34f6cbf4ebb5752875b5961f7e/Lib/test/test_socket.py#L5145-L5146

Reply:
Thanks.
The call is removed.